tests/terraform/checks/resource/gcp/test_GoogleComputeFirewallUnrestrictedIngress3306.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+import hcl2
+
+from checkov.terraform.checks.resource.gcp.GoogleComputeFirewallUnrestrictedIngress3306 import check, PORT
+from checkov.common.models.enums import CheckResult
+
+
+class TestGoogleComputeFirewallUnrestrictedIngress3306(unittest.TestCase):
```

Comment:
please adjust the tests to use a real Terraform file, like here, thanks! 
https://github.com/bridgecrewio/checkov/blob/master/tests/terraform/checks/resource/gcp/example_GoogleComputeFirewallUnrestrictedIngress22/main.tf
https://github.com/bridgecrewio/checkov/blob/master/tests/terraform/checks/resource/gcp/test_GoogleComputeFirewallUnrestrictedIngress22.py

Reply:
I can do that, but it does not match the guidance listed for [Contributing Python-based Custom Policies](https://www.checkov.io/6.Contribution/Contribute%20Python-Based%20Policies.html) as far as I can tell. The example test case provides the HCL inline and reads from that versus referencing an external file.

Reply:
🙈 thanks for pointing out the not up-to-date docs, I will adjust them in the next days

Reply:
All good! Please take a look at the new test cases that should match your request: 

https://github.com/bridgecrewio/checkov/pull/2430/commits/68f37bada0a8c67eb15a6909147d0432e9088a21